_includes/layout/base/html-head.html

```diff
@@ -16,6 +16,8 @@
 {% if page.lang == 'bg' or page.lang == 'el' or page.lang == 'ko' or page.lang == 'hi' or page.lang == 'pl' or page.lang == 'sl' or page.lang == 'ro' or page.lang == 'ru' or page.lang == 'tr' or page.lang == 'uk' or page.lang == 'zh_CN' or page.lang == 'zh_TW' %}<link rel="stylesheet" href="/css/sans.css">{% endif %}
 <script type="text/javascript" src="/js/base.js"></script>
 {% if page.id != 'download' %}<script type="text/javascript" src="/js/main.js"></script>{% endif %}
+<script src="/js/jquery/jquery-1.11.2.min.js"></script>
+<script src="/js/jquery.qrcode.min.js"></script>
```

Comment:
QR code is regenerated anytime a user selects a sum, and/or enters a message. This can be removed, however there will be no need for quick donation buttons, and for neither custom amount input, nor optional message input. In the end the donation window will contain only static QR code and address.

If this is what we want to show people - a boring modal window, then jQuery and QR code generation can be removed and appropriate code.